_posts/2019-06-24-qsharp-triple-slash.md

```diff
@@ -70,3 +70,8 @@ between sections or input parameter subsections when it generates the stub.
 
 If you're a Q# developer, consider giving it a try. It's very low overhead and totally free. Hopefully, it will make your life a little
 bit easier while documenting your Q# code.
+
+*Approved for Public Release; Distribution Unlimited. Public Release Case Number 19-2197.
+The author�s affiliation with The MITRE Corporation is provided for identification purposes only and is not intended to convey or imply
+MITRE�s concurrence with, or support for, the positions, opinions or viewpoints expressed by the author.
+� 2019 The MITRE Corporation. All Rights Reserved.*
```

Comment:
Would it be possible to remove this line? The repo is under MIT and I am not sure if this copyright is compatible with that.

Reply:
I'll ask the powers that be on this end. Are you just referring to the last line with the copyright statement, or to the entire disclaimer?

Reply:
Just the last line, the rest of it is fine.

Reply:
Sorry @jclapis I realize it has been a while, but that I never got this PR closed! Is there anything I can do to help on this?